src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx

```diff
@@ -303,10 +307,12 @@ const ArtworkDetail = ({ size = "small", label, value }: ArtworkDetailProps) =>
 }
 
 const artworkContentQuery = graphql`
-  query ArtworkContentQuery($slug: String!) {
+  query ArtworkContentQuery($slug: String!, $imageSize: Int!) {
     artwork(id: $slug) {
       image {
-        url
+        resized(height: $imageSize, version: "normalized") {
```

Comment:
**question**: We are calculating for `width` and passing it for height. Did you try passing it for width. I mean - `resized(width: $imageSize, version: "normalized")`

Reply:
Good point, I've updated that to use the window height instead

 
![Screen Shot 2022-09-01 at 14 24 43](https://user-images.githubusercontent.com/36475005/187913132-a71bb181-da8d-4317-8a83-02a230f338fc.png)

Reply:
Pretty sure we want to use windowWidth here, not windowHeight

Reply:
It's line 313 that should change here: 
```suggestion
        resized(width: $imageSize, version: "normalized") {
```